services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java

```diff
@@ -169,12 +171,12 @@ public boolean isError() {
    * @return error type.
    */
   public int errorType() {
-    if (!isError()) {
+    final String errorType = headers.get(ERROR_TYPE);
+    if (errorType == null) {
       throw new IllegalStateException("Message is not an error");
     }
```

Comment:
Let's start returning `-1`. Let's change contract. I regret about this `throw new IllegalSt...`.